perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -62,17 +62,41 @@ def Run(benchmark_spec):
     A list of sample.Sample objects.
   """
   vms = benchmark_spec.vms
-  if not vms[0].IsReachable(vms[1]):
-    logging.warn('%s is not reachable from %s', vms[1], vms[0])
+  results = []
+  for origin_index in [0, 1]:
```

Comment:
I made the change as you suggested.  I'm not sure how to add the check in CheckPrerequisites, though.  In the examples I see, CheckPrerequisites takes no arguments and it does check the value of the flag num_vms.   But if someone has a YAML config file with more more than two vms, will that cause the num_vms flags to change?

I could do the VM count check in the Prepare method instead of the CheckPrerequisites method.